core/src/main/java/org/mule/runtime/core/internal/processor/simple/ParseTemplateProcessor.java

```diff
@@ -112,6 +116,8 @@ private MediaType buildMediaType() {
 
   @Override
   public CoreEvent process(CoreEvent event) {
+    evaluateCorrectArguments();
```

Comment:
can't this be checked on initialize? or does it cause problems

Reply:
it breaks backwards compatibility. THat's why the test broke